### DISCUSSION TITLE
Update CertificateAuthenticationOptions.CustomTrustStore once the application is running

### DISCUSSION THREAD

**Discussion by aalsamoht** [dotnet/aspnetcore#29682] [Q&A]
I might be missing something obvious, but is it possible to update or replace the CustomTrustStore after ConfigureService?
My roots and intermediates will be updated after the application has started and I would like to pass those changes along to the CertificateAuthenticationOptions.

**Comment by devbeard** [comment]
Just adding my vote for the question. This is a common requirement for applications that run while cert stores or configurations change.
